[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //div[@class="ivu-select-head-flex"]/input
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //li[text()="金属（演示）"]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //button[contains(@class, "ivu-btn-primary")]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="系统管理"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="系统设置"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="用户权限管理"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="角色管理"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//table[@class="vxe-table--body"])[1]//tr/td[2]//span[text()="1测试角色代码2"]
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D1CBDC7B20>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c9e7f8357ebf8417e8cc0999076193be
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D1CBDC72B0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c9e7f8357ebf8417e8cc0999076193be
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D1CBDC7550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c9e7f8357ebf8417e8cc0999076193be
[32mINFO    [0m root:driver_manager.py:56 🛑 已从实例池移除并关闭 driver：2000572022160